src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +81,14 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Returns an modifiable java List view with arrangeable elements
```

Comment:
The header comment does not follow the convention. All non trivial methods should have javadoc format header comments. (First line is the description of method, followed by @param for specifying the parameters and @return for specifying the return values, if applicable)